.pfnci/script.sh

```diff
@@ -139,6 +148,26 @@ prepare_docker() {
   run gcloud auth configure-docker
 }
 
+# is_known_base_branch returns 0 only if the given branch name is a known
+# base development branch.
+is_known_base_branch() {
+  local branch="${1##refs/heads/}"
+  for BASE_BRANCH in master v6; do
+    if [ "${branch}" = "${BASE_BRANCH}" ]; then
+      return 0
+    fi
+  done
+  return 1
+}
+
+# get_base_branch returns the base development branch for the current HEAD.
+get_base_branch() {
+  for BASE_BRANCH in master v6; do
+    run git merge-base --is-ancestor "${BASE_BRANCH}" HEAD && echo "${BASE_BRANCH}" && return 0
```

Comment:
I'm not sure but I can't run above command so maybe we need this:
```suggestion
    run git merge-base --is-ancestor "origin/${BASE_BRANCH}" HEAD && echo "${BASE_BRANCH}" && return 0
```